client/src/Reviews/AddReview.jsx

```diff
@@ -400,13 +456,13 @@ const AddReview = ({ product, chars, ratings, setShowForm }) => {
                 onChange={handleEmailChange}
                 required
               />
-            </Styles.flexContainerCol>
-          </Styles.section>
+            </Styles.flexFitCol>
+          </Styles.flexWidth>
 
-          <Styles.flexContainerShort>
+          <Styles.flexFit>
             <Submit reviewInfo={reviewInfo} ratings={ratings} />
```

Comment:
When I tried submitting a review, it was telling me I hadn't given an Overall rating when I did click on the stars. The state for userRating does properly update with the handleStarEvent handler but the rating never gets put in the reviewInfo state before it gets passed to <Submit/>